pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -277,25 +312,33 @@ func (sync *Synchronizer) handler(msg proto.Message) error {
 			return client.ErrStopReading
 		}
 	case *gnmi.SubscribeResponse_Update:
+		log.Info("Response", v)
```

Comment:
do you need to print the whole response?

Reply:
removed entirely, for debugging purposes